recipes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "spyder-unittest" %}
+{% set version = "0.1b2" %}
```

Comment:
Normally we prefer formal releases. However if there is nothing other than a beta, I think we can make an exception. @conda-forge/core?